File: server/models/choice.js
const mongoose = require('mongoose');

let choiceSchema = new mongoose.Schema({
    rice:{
        type: String,
        required: true
    },
    ricePrice:{
        type: Number,
        required: true
    },
    topping:{
        type: String,
        required: true
    },
    toppingPrice:{
        type: Number,
        required: true
    }

});

module.exports = mongoose.model("choice",choiceSchema,"choice_data");

File: lib/models/choice.dart
class Choice{
  final String rice;
  final double ricePrice;
  final String topping;
  final double toppingPrice;

  

  factory Choice.fromJson(Map<String,dynamic> json){
    return Choice(
      rice: json['rice'],
      ricePrice: json['ricePrice'],
      topping: json['topping'], 
      toppingPrice:json['toppingPrice']
      );
  }

  Choice({required this.rice, required this.topping, required this.ricePrice, required this.toppingPrice});
}

